src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/AdjudicationMigrateDto.kt

```diff
@@ -60,3 +64,13 @@ data class MigrateAssociate(
   @Schema(description = "associate prisoner number")
   val associatedPrisoner: String?,
 )
+
+@Schema(description = "reporting officer")
+data class ReportingOfficer(
+  @Schema(description = "reporting officer username")
+  val username: String,
+)
+
+enum class NomisGender {
+  M, F, NK, NS, REF
```

Comment:
a bit risky redefining gender outside of nomis - but is it's there just to support migration should be fine